LICENSE

```diff
@@ -1,25 +1,18 @@
 Copyright 2017 - 2018 DiffBlue Ltd.
```

Comment:
I suppose this should actually be Diffblue with a lower case b, while we're here